### ISSUE TITLE
ResolutionError( pkg_resources.ResolutionError: Script 'scripts/captiveflask' not found

### ISSUE THREAD

**Issue by CRAZYR00T** [P0cL4bs/wifipumpkin3#125]
Good Evening, im having a error when i choose captive flask proxy, basically when i choose any proxy. I will describe this with screenshots.

**To Reproduce**
1. `sudo wifipumpkin3`
2. `proxies`
3. `set proxy captiveflask`
4. `set captiveflask.Login_v4 true`
5. `start`

**Expected behavior**

(IMAGE TAKEN FROM A VIDEO)

![transferir](https://user-images.githubusercontent.com/28877800/105243947-a997f780-5b67-11eb-8aad-3a2ff0ae197e.png)

**Screenshots**

**ERROR:**

`[  captiveflask  ] 16:32:51  - Traceback (most recent call last):
  File "/usr/local/bin/captiveflask", line 4, in <module>
    __import__('pkg_resources').run_script('wifipumpkin3==1.0.8', 'captiveflask')
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 650, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1434, in run_script
    raise ResolutionError(
pkg_resources.ResolutionError: Script 'scripts/captiveflask' not found in metadata at '/usr/local/lib/python3.9/dist-packages/wifipumpkin3-1.0.8.dist-info'
`

![transferir (1)](https://user-images.githubusercontent.com/28877800/105243730-5d4cb780-5b67-11eb-929b-d193a97f76f4.png)

**Desktop (please complete the following information):**
* Wifipumpkin3 version you are using ( `wifipumpkin3 --version `). `wifipumpkin3 v1.0.8 branch: dev`
* Virtual Machine (yes or no and which):  **Yes**
* Operating System: **Kali linux**
* version: **_2020.4_**

**Comment by mh4x0f** [comment]
try this on directory of `wifipympkin3`:
``` bash
sudo make clean
sudo pip3 uninstall wifipumpkin3==1.0.8
sudo python3.7 setup.py install
```

**Comment by CRAZYR00T** [comment]
I tryed do `sudo make clean` and this appears: 

`rm -rf build dist README MANIFEST *.egg-info
python3.7 setup.py clean --all
running clean
'build/lib' does not exist -- can't clean it
'build/bdist.linux-x86_64' does not exist -- can't clean it
'build/scripts-3.7' does not exist -- can't clean it
`
Next i uninstalled wifipumpkin3==1.0.8 and tryed do `sudo python3.7 setup.py install
` and this is what happened:

![transferir (2)](https://user-images.githubusercontent.com/28877800/105355128-86159100-5be9-11eb-99fe-d3ed6c03b1cb.png)

Basically i can't install wifipumpkin3 with python3.7.. some solutions?

**Comment by mh4x0f** [comment]
try with python3.9:
``` sh 
rm -rf build dist README MANIFEST *.egg-info
python3.9 setup.py clean --all
sudo pip3 uninstall wifipumpkin3==1.0.8
sudo python3.9 setup.py install
```

**Comment by CRAZYR00T** [comment]
Thank you, solved the problem!
